server/services/utils/checkPriceThenUpdateDb.js

```diff
@@ -0,0 +1,22 @@
+const axios = require("axios");
+
+const checkPriceChange = (currentPrice, fetchedPrice, productId, name) => {
+  console.log(`Product Name ${name}`);
+  console.log(`Current Price ${currentPrice}`);
+  console.log(`Fetched Price ${fetchedPrice}`);
+
```

Comment:
Remove these if they are for testing.

Reply:
Placed logs for visual purposes to view the progress of the scraper, might have to remove all console logs on future deployment